.env.example

```
@@ -18,6 +18,8 @@ PROTO_DESCRIPTOR_PATH="./provers/celestia-prover/proto_descriptor.bin"
 # RPC of a celestia-node (light, bridge, or full)
 export CELESTIA_NODE_URL="http://localhost:52351"
 # Auth token for the celestia-node
-export CELESTIA_NODE_AUTH_TOKEN=""
+export CELESTIA_NODE_AUTH_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJwdWJsaWMiLCJyZWFkIiwid3JpdGUiLCJhZG1pbiJdfQ.qxoN_a-k7aZZE77VSpfgyvI_0eNvJ_OgBsT35zxUQnQ"
```

[nit] this probably won't work for anyone else

```suggestion
export CELESTIA_NODE_AUTH_TOKEN=""
```

---

we have a da auth token hardcoded here https://github.com/celestiaorg/celestia-zkevm-ibc-demo/blob/main/testing/files/da_auth_token

---

oh wow, good find